cypress/support/api_commands.js

```diff
@@ -233,3 +238,62 @@ Cypress.Commands.add('apiEnableOpenServer', (enable = true) => {
         body: config,
     });
 });
+
+/**
+ * Creates a new user via the API, adds them to 3 teams, and sets preference to bypass tutorial.
+ * Then logs in as the user
+ @param {Object} user - Object of user email, username, and password that you can optionally set. Otherwise use default values
+ @returns {Object} Returns object containing email, username, id and password if you need it further in the test
+ */
+Cypress.Commands.add('loginAsNewUser', ({email = `newE2ETestUser${Date.now()}@mattermost.com`, username = `NewE2ETestUser${Date.now()}`, password = 'password123'} = {}) => {
```

Comment:
nit: Would be nice if the random `Date.now()` is same value for email and username

Reply:
Addressed in latest push.